src/components/molecules/WalletNetworkSwither.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { ReactElement, useState, useEffect } from 'react'
```

Comment:
small typo in filename `WalletNetworkSwither.tsx`, missing a `c`: `WalletNetworkSwitcher.tsx`